### ISSUE TITLE
Add checksum into data copying

### ISSUE THREAD

**Issue by twshimwell** [mhardcastle/ddf-pipeline#249]
Thinking might be a good idea to add some security that the data are fully copied.

**Comment by mhardcastle** [comment]
Agree, it would be, but the right way to do this is have the service at the far end confirm that it agrees about the checksum of the data. So that needs an interface to upload more sophisticated than rsync, most likely.

(Rsync can verify that a copy is a copy by using checksums, but that would slow the process down considerably if the checksum drops.)
